cognite_toolkit/_cdf_tk/prototypes/init.py

```diff
@@ -0,0 +1,250 @@
+        ("instruction", ""),  # user instructions for select, rawselect, checkbox
+        ("text", ""),  # plain text
+        ("disabled", "fg:#858585 italic"),  # disabled choices for select and checkbox prompts
+    ]
+)
+
+INDENT = "  "
+POINTER = INDENT + "▶"
+
+
+class Packages(dict, MutableMapping[str, dict[str, Any]]):
+    @classmethod
+    def load(cls) -> Packages:
+        packages = {}
+        for module in _packages.__all__:
+            manifest = Path(_packages.__file__).parent / module / "manifest.yaml"
+            if not manifest.exists():
+                continue
+            content = manifest.read_text()
+            if yaml.__with_libyaml__:
+                packages[manifest.parent.name] = yaml.CSafeLoader(content).get_data()
+            else:
+                packages[manifest.parent.name] = yaml.SafeLoader(content).get_data()
+        return cls(packages)
+
+
+class InitCommand(ToolkitCommand):
+    def _build_tree(self, item: dict | list, tree: Tree) -> None:
+        if not isinstance(item, dict):
+            return
+        for key, value in item.items():
+            subtree = tree.add(key)
+            for subvalue in value:
+                if isinstance(subvalue, dict):
+                    self._build_tree(subvalue, subtree)
+                else:
+                    subtree.add(subvalue)
+
+    def _create(
+        self, init_dir: str, selected: dict[str, dict[str, Any]], environments: list[str], mode: str | None
+    ) -> None:
+        if mode == "overwrite":
+            print(f"{INDENT}[yellow]Clearing directory[/]")
+            if Path.is_dir(Path(init_dir)):
+                shutil.rmtree(init_dir)
+
+        modules_root_dir = Path(init_dir) / "modules"
+        modules_root_dir.mkdir(parents=True, exist_ok=True)
+
+        for package, modules in selected.items():
+            print(f"{INDENT}[{'yellow' if mode == 'overwrite' else 'green'}]Creating {package}[/]")
+
+            # package_dir = modules_dir / package
+
+            for module in modules:
+                print(f"{INDENT*2}[{'yellow' if mode == 'overwrite' else 'green'}]Creating module {module}[/]")
+                source_dir = Path(_packages.__file__).parent / package / module
+                if not Path(source_dir).exists():
+                    print(f"{INDENT*3}[red]Module {module} not found in package {package}. Skipping...[/]")
+                    continue
+                module_dir = modules_root_dir / package / module
+                if Path(module_dir).exists() and mode == "update":
+                    if questionary.confirm(
+                        f"{INDENT}Module {module} already exists in folder {module_dir}. Would you like to overwrite?",
+                        default=False,
+                    ).ask():
+                        shutil.rmtree(module_dir)
+                    else:
+                        continue
+
+                shutil.copytree(source_dir, module_dir)
```

Comment:
This copies everything, we need to skip the `default.config.yaml` as these are part of the specification of the template, but not the template itself.
![image](https://github.com/cognitedata/toolkit/assets/60234212/370cb280-d619-4384-9a23-4c8a135fe724)